clients/highwater/highwater.go

```diff
@@ -9,7 +9,7 @@ import (
 	"path"
```

Comment:
As we do many changes not related to the scope of this PR (like removing hakken/renaming module name/adding logger to the server object), why do we keep a highwater client ?

Reply:
Good question, should I remove it ?